app/.gitlab-ci.yml

```diff
@@ -512,6 +512,20 @@ release:frontend:
       changes:
       - app/**/*
 
+release:frontend-sourcemaps:
+  needs: ["build:frontend"]
+  stage: release
+  image: $FRONTEND_NEXT_TAG
+  inherit:
+    # no docker login
+    default: false
+  script:
+    - cd /app && SENTRY_AUTH_TOKEN=$SENTRY_AUTH_TOKEN yarn upload-sourcemap
+  rules:
+    - if: ($DO_CHECKS == "true") && ($CI_COMMIT_BRANCH == $RELEASE_BRANCH)
```

Comment:
This should have the same checks as the others, just `- if: $CI_COMMIT_BRANCH == $RELEASE_BRANCH`